pkg/rendering/helm_test.go

```diff
@@ -0,0 +1,79 @@
+package rendering_test
+
+import (
+	. "code.cloudfoundry.org/quarks-secret/pkg/rendering"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"helm.sh/helm/v3/pkg/chart"
+)
+
+var _ = Describe("Rendering", func() {
+	var (
+		engine Engine
+	)
+	values := map[string]interface{}{"outer": "DEFAULT", "inner": "DEFAULT"}
+	defaults := map[string]interface{}{
+		"Values": map[string]interface{}{
+			"outer": "spouter",
+			"inner": "inn",
+			"global": map[string]interface{}{
+				"callme": "Ishmael",
+			},
+		},
+	}
+	BeforeEach(func() {
+		engine = NewHelmRenderingEngine()
+	})
+	Describe("Using helm engine", func() {
+
+		It("render templates correctly", func() {
+
+			helm := engine.(HelmRenderingEngine)
+			files := []*chart.File{
+				{Name: "templates/test1", Data: []byte("{{.Values.outer | title }} {{.Values.inner | title}}")},
```

Comment:
Our code never uses more than one file

Reply:
Yes, but I think it's fine to keep it for testing the rendering engine (what you can, and what you can't do with the helm templating). So there are no surprises when using the templating system. You can just quickly test what we support and what we don't (as we will use more engines in the future as far as I understood)

Reply:
I kept the intention of the unit test, without exposing unused functionality.